sda-svc/templates/auth-deploy.yaml

```
@@ -53,7 +53,7 @@ spec:
 {{- end }}
         env:
         - name: LOG_LEVEL
-          value: {{ upper .Values.global.logLevel }}
+          value: {{ upper .Values.global.log }}
```

```suggestion
          value: {{ upper .Values.global.logLevel }}
```
As per vaules file this should be `logLevel`

---

https://github.com/neicnordic/sda-helm/blob/e8fa0498dd1eff9bafdc797d82728f5088798c92/.github/ci_tests/s3.yaml#L52

---

That explains why you don't see detailed logs in the tests.

This is how the templates look like:
```
      {{- if .Values.global.logLevel }}
        - name: LOG_LEVEL
          value: {{ .Values.global.logLevel | quote }}
      {{- end }}
```

---

Fixed